src/svgdom/length.hpp

```diff
@@ -48,7 +48,8 @@ enum class length_unit{
 	in,
 	pt,
 	pc,
-	mm
+	mm,
+	dip
```

Comment:
Please add a comment here explaining that `dip` is a unit introduced in Sciter, and provide link to relevant info.